modules/ROOT/pages/setup.adoc

```diff
@@ -79,16 +83,20 @@ wrapper.java.maxmemory=4606
 
 ----
 
+. Install your license. See https://docs.mulesoft.com/mule-runtime/4.2/installing-an-enterprise-license[Install an Enterprise License] for more information.
```

Comment:
This should also be muscle memory by now: limit steps and list items to no more than one sentence.
```suggestion
. Install your license. 
See https://docs.mulesoft.com/mule-runtime/4.2/installing-an-enterprise-license[Install an Enterprise License] for more information.
```

Reply:
Fixed.